README.md

```diff
@@ -2,13 +2,62 @@
 
 [![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/39253)
 
-CFD trading on Bitcoin.
+CFD trading on Bitcoin - non-custodial, peer-to-peer, Bitcoin only.
 
-Details coming soon.
+Trading on ItchySats is secured by DLCs (Discreet Log Contracts).
+Here is some material to read up on DLCs and ItchySat's protocol implementation:
 
-## Quickstart
+- [The ItchySats Roadmap](https://itchysats.medium.com/itchysats-roadmap-to-the-most-awesome-bitcoin-dex-464a42bf4881) which includes a simplified example how the protocol works
+- [Blogpost](https://comit.network/blog/2022/01/11/cfd-protocol-explained) by @luckysory of the COMIT team that describing the protocol in detail
+- [DLC overview](https://bitcoinops.org/en/topics/discreet-log-contracts/) on Bitcoin Optech for information beyond ItchySats
 
-All the components can be started at once by running the following script:
+## Quickstart: Users / Traders
+
+This guide is for how to use ItchySats as a taker.
+Currently, ItchySats does not officially support multiple makers yet.
```

Comment:
"Currently" and "yet" are redundant here.

```suggestion
Currently, ItchySats does not officially support multiple makers.
```

Reply:
Really? I feel the `yet` emphasizes that it will be supported in the future, whereas the `current` just indicates that it is not supported now, but does not tell anything about the future.

Reply:
You don't have to take this one! But I think "currently" implies that "not-currently" (in the future) this could change.

Reply:
"Currently" to me implies that it may change in the future, otherwise you could just say: "Itchysats does not support multiple makers." if it were a permanent goal.

:man_shrugging: